schemas/alpha/azure.yaml.json

```diff
@@ -1240,9 +1222,8 @@
                     "title": "Type of resource",
                     "description": "The type of resource to be created. (Example: db.postgres)",
                     "enum": [
-                        "db.postgres",
```

Comment:
The description should be updated. Don't use db.postgres as example.

Reply:
Updated.